requirements.txt

```diff
@@ -1,7 +1,14 @@
-httmock
-wooper
-PyRTF3
-xhtml2pdf
--e .
-git+git://github.com/superdesk/superdesk-planning.git@release/2.0#egg=superdesk-planning
-git+git://github.com/superdesk/superdesk-core.git@v2.0#egg=superdesk-core
+Babel>=2.5.3,<3.0
+WTForms==2.2.1
+flask-webpack>=0.1.0,<0.2
+Flask-WTF>=0.14.2,<0.15
+flask-limiter>=0.9.5.1,<0.9.6
+Flask-Caching>=1.9.0
+flask_pymongo>=0.5.2,<1.0
+honcho>=1.0.1
+gunicorn>=19.7.1
+icalendar>=4.0.3,<4.1
+PyRTF3>=0.47.5
+xhtml2pdf>=0.2.4
+superdesk-core==2.0.10
+superdesk-planning@https://github.com/superdesk/superdesk-planning/archive/v2.0.1.zip
```

Comment:
Is there a reason to use github here instead of pypi? i.e. `superdesk-planning==2.0.2` should work right?

Reply:
Hi Mark,
No reason, I think at the time I was writing this file planning 2.0 was not on pypy yet.
I will change it.